app/controllers/bookings_controller.rb

```diff
@@ -9,18 +9,14 @@ def new
     @tool = Tool.find(params[:tool_id])
   end
 
+
   def create
     @tool = Tool.find(params[:tool_id])
     @booking = Booking.new(booking_params)
     @booking.tool = @tool
+    @booking_cost = @booking.booking_cost
```

Comment:
i dont really understand what this line does? also - 'booking_cost' is a bit of a messy column name, 'total_cost' would have been better - we should make a note of this for next week.